core/templates/domain/skill/skill-creation-backend-api.service.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Backend service for creating a new skills
+ */
+
+export interface IRubricBackend {
+  'difficulty':string,
```

Comment:
Why are you using single quotes here and elsewhere?

Reply:
I was working on skillObjectMigration. when I use the properties without a single quote it was showing me an error to use quotes for the underscore properties, there I didn't realize that the properties without the underscore don't need the single quotes
thanks for pointing this out 
I'll change them to without quotes onne